packages/build/src/release.ts

```diff
@@ -54,12 +54,6 @@ export async function release(
     redactConfig(config)
   );
 
-  if (command === 'bump') {
-    // updates the version of internal packages to reflect the tagged one
-    await bumpNpmPackages(config.version);
-    return;
-  }
```

Comment:
This release process should still involve a bump of the versions of all packages that the `@mongosh/cli-repl` package depends on, right?

Reply:
Hm right, was going to make this part of a script but re-added it as a function which calls the `bump-packages`.

So an idea could be:
We bump all other packages with `bump-packages` at release except `mongosh` and `@mongosh/cli-repl` which will always be excluded from `bump-packages`.
For `mongosh` and `@mongosh/cli-repl`, the version is manually set at `config.version` from release _before_ the rest of the packages get bumped.

Would this make sense with what we're going for?